io.github.saeugetier.photobooth.json

```
@@ -0,0 +1,106 @@
+{
+    "app-id": "io.github.saeugetier.photobooth",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.9",
+    "sdk": "org.kde.Sdk",
+    "command": "qtbooth",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=home",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "opencv4",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DBUILD_TESTS=OFF",
+                "-DBUILD_PERF_TESTS=OFF",
+                "-DBUILD_DOCS=OFF",
+                "-DBUILD_LIST=imgproc,imgcodecs",
+                "-DOPENCV_GENERATE_PKGCONFIG=ON",
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DOPENCV_LIB_INSTALL_PATH=/app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/opencv/opencv/archive/4.5.5.tar.gz",
+                    "sha256": "a1cfdcf6619387ca9e232687504da996aaa9f7b5689986b8331ec02cb61d28ad"
+                }
+            ],
+            "cleanup": [
+                "/bin/*",
+                "/share/opencv4/*"
+            ]
+        },
+        {
+            "name": "onnxruntime",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/include ${FLATPAK_DEST}/lib",
+                "cp -r include/* ${FLATPAK_DEST}/include",
+                "cp -r lib/* ${FLATPAK_DEST}/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/microsoft/onnxruntime/releases/download/v1.21.0/onnxruntime-linux-x64-1.21.0.tgz",
+                    "sha256": "7485c7e7aac6501b27e353dcbe068e45c61ab51fbaf598d13970dfae669d20bf"
+                },
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "aarch64"
+                    ],
+                    "url": "https://github.com/microsoft/onnxruntime/releases/download/v1.21.0/onnxruntime-linux-aarch64-1.21.0.tgz",
+                    "sha256": "4508084bde1232ee1ab4b6fad2155be0ea2ccab1c1aae9910ddb3fb68a60805e"
+                }
+            ]
+        },
+        {
+            "name": "qtvirtualkeyboard",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
```

```suggestion
                "-DCMAKE_BUILD_TYPE=Release"
```

this is also done automatically